pxf/pxf-hbase/src/main/java/org/apache/hawq/pxf/plugins/hbase/HBaseFilterBuilder.java

```diff
@@ -165,6 +165,14 @@ private Filter handleSimpleOperations(FilterParser.Operation opId,
         ByteArrayComparable comparator = getComparator(hbaseColumn.columnTypeCode(),
                 constant.constant());
 
+        if(operatorsMap.get(opId) == null){
+            //HBase not support HDOP_LIKE, use 'NOT NULL' Comarator
```

Comment:
No, @hsyuan comment was that the comment should read "//HBase does not support..." HBase does not support the LIKE Filter as far as I am aware